net/tgt/files/etc/init.d/tgt

```diff
@@ -164,8 +159,8 @@ handle_target() {
 			return 1
 		}
 	done
-	config_foreach handle_lun lun $tgtid || return 1
-	config_foreach bind_account_to_target account $tgtid || return 1
+	config_foreach validate_lun_section lun handle_lun || return 1
+	config_foreach validate_account_section account bind_account_to_target || return 1
```

Comment:
Just wanted to mention here that, because of dynamic scope, `$tgtid` is available to `handle_lun()` and `bind_account_to_target()` without being explicitly passed to them as arguments.